zeppelin-web/src/app/helium/helium.html

```diff
@@ -44,30 +61,54 @@ <h3 class="new_h3">
 </div>
 
 <div class="box width-full heliumPackageContainer">
+  <div class="row"
+       style="padding-bottom: 15px"
+       ng-if="getPackageSize(defaultPackages, pkgListByType) === 0">
+    <div class="col-md-12 gray40-message">
+      <em>Currently there is no available package to be listed</em>
+    </div>
+  </div>
   <div class="row heliumPackageList"
-       ng-repeat="(pkgName, pkgSearchResult) in defaultPackages">
-
+       ng-repeat="(pkgName, pkgSearchResult) in defaultPackages"
+       ng-show="$parent.pkgListByType == pkgSearchResult.pkg.type">
     <div class="col-md-12">
       <div class="heliumPackageHead">
         <div class="heliumPackageIcon"
+             ng-if="pkgSearchResult.pkg.type !== 'INTERPRETER'"
              ng-bind-html=pkgSearchResult.pkg.icon></div>
+        <div class="heliumPackageIcon"
+             ng-if="pkgSearchResult.pkg.type === 'INTERPRETER'"
+             ng-bind-html=intpDefaultIcon></div>
         <div class="heliumPackageName">
           <span ng-if="hasNpmLink(pkgSearchResult)">
             <a target="_blank" href="https://www.npmjs.com/package/{{pkgName}}">{{pkgName}}</a>
           </span>
-          <span ng-if="!hasNpmLink(pkgSearchResult)" ng-class="{'heliumLocalPackage': isLocalPackage(pkgSearchResult)}">
+          <span ng-if="!hasNpmLink(pkgSearchResult) && !hasMavenLink(pkgSearchResult)" ng-class="{'heliumLocalPackage': isLocalPackage(pkgSearchResult)}">
             {{pkgName}}
           </span>
+          <span ng-if="hasMavenLink(pkgSearchResult)">
+            <a target="_blank"
+               href="http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22{{pkgSearchResult.pkg.artifact.split('@')[0]}}%22%20AND%20v%3A%22{{pkgSearchResult.pkg.artifact.split('@')[1]}}%22">
+              {{pkgName}}
+            </a>
+          </span>
           <span class="heliumType">{{pkgSearchResult.pkg.type}}</span>
         </div>
         <div ng-show="!pkgSearchResult.enabled"
              ng-click="enable(pkgName, pkgSearchResult.pkg.artifact)"
+             ng-if="pkgSearchResult.pkg.type !== 'INTERPRETER'"
              class="btn btn-success btn-xs"
              style="float:right">Enable</div>
         <div ng-show="pkgSearchResult.enabled"
              ng-click="disable(pkgName)"
+             ng-if="pkgSearchResult.pkg.type !== 'INTERPRETER'"
              class="btn btn-info btn-xs"
              style="float:right">Disable</div>
+        <div ng-show="!pkgSearchResult.enabled"
+             ng-click="enable(pkgName, pkgSearchResult.pkg.artifact)"
+             ng-if="pkgSearchResult.pkg.type === 'INTERPRETER'"
+             class="btn btn-success disabled btn-xs"
+             style="float:right">Coming soon</div>
```

Comment:
How about make this button click-able and show dialog that just has instructions for installing interpreter manually using bin/install-interpreter.sh ?